dep/nginx-sites.sh

```diff
@@ -36,6 +36,14 @@ createLogDir() {
         mkdir -p "$1"/logs
     fi
 }
+clearLogFiles() {
+    logdir="$1/logs"
+    for logfile in error.nginx.log access.nginx.log; do
+        if [ -f "$logdir/$logfile" ]; then
+            > "$logdir/$logfile"
```

Comment:
@yfleuren Zou geen ramp zijn, maar het lijkt me net iets netter om de laatste 1000 regels te behouden, zodat in ieder geval je error log niet onverwacht leeg is als je moet hetstarten/reloaden.

Het zou dan zoiets worden:

```
clearLogFiles() {
    logdir="$1/logs"
    keep_lines=1000
    
    for logfile in error.nginx.log access.nginx.log; do
        if [ -f "$logdir/$logfile" ]; then
            # Check of file groot genoeg is om te truncaten
            if [ $(wc -l < "$logdir/$logfile") -gt $keep_lines ]; then
                tail -n $keep_lines "$logdir/$logfile" > "$logdir/$logfile.tmp"
                mv "$logdir/$logfile.tmp" "$logdir/$logfile"
            fi
        fi
    done
}
```